### ISSUE TITLE
MacOS Wallet: Reden-Qt cannot be opened because of a problem.

### ISSUE THREAD

**Issue by skydivematy** [RedenCore/Reden#33]
Any Ideas???
On Macbook Pro 2012 - MacOs Sierra 10.12... it says my McOs version is to old so i upgrade to 10.13.05 and now it says Reden-Qt cannot be opened because of a problem.
Check with the developer to make sure Reden-Qt works with this version of macOS. You may need to reinstall the application. Be sure to install any available updates for the application and macOS.

Click Report to see more detailed information and send a report to Apple.

**Comment by thebeardbe** [comment]
Yes, the reason for this is that in OS X the boost library is not installed.

Try installing the boost library through brew and you should be ok.

If you have homebrew installed do

`brew install boost`

to install homebrew go to [https://brew.sh/](https://brew.sh/)

**Comment by skydivematy** [comment]
Hi thebeardbe,
thanks for this info.
It works.....
